torch/csrc/jit/runtime/operator.h

```diff
@@ -160,6 +160,16 @@ struct TORCH_API Operator {
         });
   }
 
+  std::vector<at::Tag> getTags() const {
+    return op_.fold<std::vector<at::Tag>>(
```

Comment:
You sure you want a copy here? ArrayRef would work well here

Reply:
Yeah because we return an empty vector for JitOnlyOperators. c10::ArrayRef wouldn't work because we return a newly created empty array/vector

Reply:
You can return an empty array ref without actually having a newly created vector, since the pointer in that case can be null